server/controller/common/getBlogs.js

```diff
@@ -0,0 +1,16 @@
+const { getBlogs } = require('../../database/queries');
+const boomify = require('../../utilis/boomify');
+
+const getAllBlogs = async (req, res, next) => {
+  try {
+    const { rows } = await getBlogs();
+    if (!rows) {
+      throw boomify(500, 'Internal Server Error');
```

Comment:
- I think when there is no blog the message will be `no blogs` not `internal server error`
- For the status code, I think `204` no content is acceptable